docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -1,30 +1,59 @@
-ms.date: 09/24/2020
+ms.date: 02/01/2021
 localization_priority: Normal
 ---
 
 # Sideload Office Add-ins in Office on the web for testing
 
-You can install an Office Add-in for testing without having to first put it in an add-in catalog by using sideloading. Sideloading can be done in either Microsoft 365 or Office on the web. The procedure is slightly different for the two platforms.
+When you sideload an add-in, you're able to install the add-in without first putting it in the add-in catalog. This is useful when testing and developing your add-in because you can see how your add-in will appear and function.
 
-When you sideload an add-in, the add-in manifest is stored in the browser's local storage, so if you clear the browser's cache, or switch to a different browser, you have to sideload the add-in again.
+When you sideload an add-in, the add-in's manifest is stored in the browser's local storage, so if you clear the browser's cache, or switch to a different browser, you have to sideload the add-in again.
+
+Sideloading can be done in either Microsoft 365 or Office on the web. The procedure is slightly different for the two platforms.
 
 > [!NOTE]
-> Sideloading as described in this article is supported on Word, Excel, and PowerPoint. To sideload an Outlook add-in, see [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md).
+> Sideloading as described in this article is supported on Excel, OneNote, PowerPoint and Word. To sideload an Outlook add-in, see [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md).
+
+## Sideload an Office Add-in in Office on the web
 
-The following video walks you through the process of sideloading your add-in in Office on the web or desktop.
+This process is supported for **Excel**, **OneNote**, **PowerPoint**, and **Word** only. For other host applications, see [the manual sideloading instructions following this section](sideload-office-add-ins-for-testing.md#sideload-an-office-add-in-in-office-on-the-web-manually). The below project assumes that you are using a project created with [the Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office).
 
-> [!VIDEO https://www.youtube.com/embed/XXsAw2UUiQo]
+1. Open [Office on the web](https://office.live.com/). Using the **Create** option, make a document in **Excel**, **OneNote**, **PowerPoint**, or **Word**. In this new document, select **Share** in the ribbon, select **Copy Link**, and copy the URL.
 
-## Sideload an Office Add-in in Office on the web
+2. In the root directory of your yo office project files, open the **package.json** file. Within the **scripts** section of this file, create a **"document"** property. Paste the URL you copied in as the value for the "document" property. For example, yours will look something like this:
+
+```json
+  "scripts": {
+    "document": "<YOUR URL>",
+    ...
+  }
+```
```

Comment:
You may need to play around with the indenting of this and the following tip to ensure they display properly and the numbering will work correctly
```suggestion
  ```json
    "scripts": {
      "document": "<YOUR URL>",
      ...
    }
  ```
```

Reply:
Ok good note - I will accept and then check